lib/adal.js

```diff
@@ -705,6 +741,7 @@ AuthenticationContext.prototype.saveTokenFromHash = function (requestInfo) {
                         }
                         this._saveItem(this.CONSTANTS.STORAGE.ACCESS_TOKEN_KEY + resource, requestInfo.parameters[this.CONSTANTS.ID_TOKEN]);
                         this._saveItem(this.CONSTANTS.STORAGE.EXPIRATION_KEY + resource, this._user.profile.exp);
+                        this.verbose('token successfully updated in local storage.');
```

Comment:
Same as above. Please update the statement to "access token updated in storage."?